Tasks/Task003.playground/Contents.swift

```diff
@@ -8,6 +8,9 @@
 //
 // Beginning of the solution
 
+let interstellarRaiting = 8.9
+let shrekRaiting  = 8.3
+let maxRaiting = max (interstellarRaiting, shrekRaiting)
```

Comment:
Как правило, мы не ставим пробел между названием функции и открывающими скобками. Рекомендую придерживаться стайлгайда https://github.com/kodecocodes/swift-style-guide
```suggestion
let maxRaiting = max(interstellarRaiting, shrekRaiting)
```